<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharacterInfoDialog</class>
 <widget class="QDialog" name="CharacterInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Status</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="11" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Attack:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Agility:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Level:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Strength:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Health:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1" alignment="Qt::AlignRight">
    <widget class="QLabel" name="attackLabel">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" alignment="Qt::AlignRight">
    <widget class="QLabel" name="levelLabel">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1" alignment="Qt::AlignRight">
    <widget class="QLabel" name="healthLabel">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Vitality:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" alignment="Qt::AlignRight">
    <widget class="QLabel" name="wisLabel">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" alignment="Qt::AlignRight">
    <widget class="QLabel" name="vitLabel">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Speed:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Dexterity:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Job:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1" alignment="Qt::AlignRight">
    <widget class="QLabel" name="speedLabel">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" alignment="Qt::AlignRight">
    <widget class="QLabel" name="dexLabel">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" alignment="Qt::AlignRight">
    <widget class="QLabel" name="jobLabel">
     <property name="text">
      <string>jobless</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>nameless</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" alignment="Qt::AlignRight">
    <widget class="QLabel" name="strLabel">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" alignment="Qt::AlignRight">
    <widget class="QLabel" name="agiLabel">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Size:</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1" alignment="Qt::AlignRight">
    <widget class="QLabel" name="sizeLabel">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Wisdom:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Energy:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1" alignment="Qt::AlignRight">
    <widget class="QLabel" name="energyLabel">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
